package.json

```diff
@@ -34,24 +34,24 @@
   },
   "dependencies": {
     "@reach/router": "^1.2.1",
-    "@types/jest": "^24.0.18",
+    "@types/jest": "^24.0.21",
     "@types/jquery": "^3.3.31",
     "@types/lodash-es": "^4.17.3",
-    "@types/reach__router": "^1.2.4",
-    "@types/react": "^16.9.2",
```

Comment:
`@types/react-dom` depends on `@types/react`, so we don't need to install `@types/react` separately. When I upgraded the packages, things broke because there ended up being two versions of `@types/react`.